test/helpers.ts

```diff
@@ -289,7 +291,7 @@ test('getCurrentCommittish', t => {
 	t.is(getCurrentCommittish(
 		'/typescript-eslint/typescript-eslint/commits/chore/lerna-4/docs/getting-started/README.md',
 		'History for docs/getting-started/README.md - typescript-eslint/typescript-eslint'
-	), 'chore'); // Wrong, but
+	), 'master'); // Uses the feedlink
```

Comment:
This only shows that the test is wrong, and thus useless.

For this to be tested correctly, it should have the correct feedLink, added and removed just for this test like we do for  `data-hotkey="t"`
```suggestion
	), 'chore/lerna-4');
	document.querySelector('the feed link')!.remove();
```

**+ the code to add it before the test**